kerchunk/intro.qmd

```diff
@@ -0,0 +1,40 @@
+---
+title: "Kerchunk"
+subtitle: References for Chunked (maybe), Compressed (maybe) N-Dimensional Arrays
+---
+
+## What is Kerchunk?
+
+Kerchunk's potential utility to the geospatial data community cannot be overstated. Kerchunk enables cloud-optimized access to traditional geospatial file formats, like NetCDF, potentially negating the need to create and store copies of data for cloud-optimized access.
```

Comment:
~Kerchunk's potential utility to the geospatial data community cannot be overstated~ maybe make a second sentence or drop.